src/Build/Xml/XmlReaderExtension.cs

```diff
@@ -71,15 +76,59 @@ public void Dispose()
             _stream?.Dispose();
         }
 
+        /// <summary>
+        /// Returns <see cref="PropertyInfo"/> of the "Normalization" internal property on the given <see cref="XmlReader"/>-derived type.
+        /// </summary>
+        private static PropertyInfo GetNormalizationPropertyInfo(Type xmlReaderType)
+        {
+            PropertyInfo propertyInfo = _normalizationPropertyInfo;
+            if (propertyInfo == null)
+            {
+                BindingFlags bindingFlags = BindingFlags.NonPublic | BindingFlags.SetProperty | BindingFlags.Instance;
+                propertyInfo = xmlReaderType.GetProperty("Normalization", bindingFlags);
+                _normalizationPropertyInfo = propertyInfo;
+            }
+            return propertyInfo;
```

Comment:
When I was on Roslyn we had a StyleCop rule that I liked, to leave an empty line after a closing curly in a statement. Now I see it everywhere :) Maybe I should propose an editorconfig change to add that, but I don't know if folks will like that.

Reply:
Done.